src/components/SearchField/SearchField.stories.mdx

```diff
@@ -0,0 +1,51 @@
+import { Meta, Preview, Props, Story } from '@storybook/addon-docs';
+import Stack from '../storyUtils/Stack';
+import SearchField from './SearchField'; import SearchFieldShowcase from './SearchFieldShowcase';
+
+<Meta title="Design System/SearchField" component={SearchField} />
+
+# TextField
+
+A universal SearchField component that the input component for any forms.
```

Comment:
this doesnt make sense when i read it

Reply:
Damn copy paste :P

Reply:
![image](https://user-images.githubusercontent.com/28539607/142386046-5e4a1470-0add-47ee-81d9-1eccf6431743.png)